### PR TITLE
fix to use the proper scale for polyline & closed graphs

### PR DESCRIPTION
**Pull request by ziktar**
I noticed that you get the wrong graph when using `graphType: "polyline"` and `closed: true` together, and it turns out the wrong axis's scaling function was being used

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /site/js/function-plot.js
-/dist
```
Comment (mauriciopoppe) [inline .gitignore:2]: The contents of `dist/` are autogenerated, maybe during the build you could copy .d.ts files from `src/` to this folder through webpack instead?

File: package.json [hunk 2]
```diff
@@ -56,7 +56,7 @@
     "d3-interpolate": "^3.0.1",
     "d3-scale": "^4.0.2",
     "d3-selection": "^3.0.0",
-    "d3-shape": "^3.1.0",
+    "d3-shape": "^2.1.0",
```
Comment (mauriciopoppe) [inline package.json:59]: Is there a reason we need to downgrade these packages?

**Review by mauriciopoppe — COMMENTED** [review]
Thanks for your contribution, I'd split the PR into one that fixes the issue you created and another one adding types.

File: package.json (hunk 2 above)
Comment (ziktar) [reply package.json:59]: No. I do for my library which has some old conflicts, but I did *not* mean for that to be included in the pull request.The change should only have the one letter change from the first commit.

**Pull request closed without merging**